lib/prefix.c

```diff
@@ -639,10 +681,39 @@ int prefix_same(const struct prefix *p1, const struct prefix *p2)
 			if (!memcmp(&p1->u.prefix_evpn, &p2->u.prefix_evpn,
 				    sizeof(struct evpn_addr)))
 				return 1;
+		if (p1->family == AF_FLOWSPEC) {
+			if (p1->u.prefix_flowspec.prefixlen !=
+			    p2->u.prefix_flowspec.prefixlen)
+				return 0;
+			if (!memcmp(&p1->u.prefix_flowspec.ptr,
+				    &p2->u.prefix_flowspec.ptr,
+				    p2->u.prefix_flowspec.prefixlen))
+				return 1;
+		}
 	}
 	return 0;
 }
 
+int prefix_opaque_cmp(const struct prefix *p1, const struct prefix *p2)
```

Comment:
shouldn't this just be part of prefix_cmp?

Reply:
done